projects/plugins/boost/composer.json

```diff
@@ -30,11 +30,11 @@
 		"yoast/phpunit-polyfills": "0.2.0"
 	},
 	"scripts": {
-		"format": [
-			"phpcbf ."
+		"fix": [
+			"composer --working-dir=../../../ phpcs:fix ./projects/plugins/boost"
 		],
 		"lint": [
-			"phpcs ."
+			"composer --working-dir=../../../ phpcs:lint ./projects/plugins/boost"
 		],
```

Comment:
Do you think that is still needed now that we have the tools at the monorepo level?

Reply:
Yes I was debating it. I think I will to discuss with @thingalon about the commands we have in Boost composer.json and package.json files. I will definitely address it in another PR.